src/components/AssetsPage/index.jsx

```diff
@@ -109,7 +80,9 @@ export class AssetsPage extends React.Component {
 }
 
 AssetsPage.propTypes = {
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
I don't think eslint is smart enough to realize that I actually am using the assetsList prop at line 41 of this file, so I have disabled the rule for that reason.